### PR TITLE
Automatic selection of Org Unit

### PR DESCRIPTION
**Pull request by zewolfe**

### CODE REVIEW

File: dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java [hunk 1]
```diff
@@ -161,6 +166,10 @@ public void onSaveInstanceState(Bundle outState) {
 
     private void loadData() {
         getLoaderManager().restartLoader(AGGREGATE_REPORT_LOADER_ID, null, this).forceLoad();
+//        pickerRecyclerViewOne.findViewHolderForAdapterPosition(0);
```
Comment (JasperTimm) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java]: Commented code! Oh no!

File: dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java [hunk 2]
```diff
@@ -210,17 +219,30 @@ public void onClick(View view) {
...
                 .findViewById(R.id.recyclerview_pickers_two);
 
+
+
+
         pickerRecyclerViewTwo.setLayoutManager(layoutManagerTwo);
         pickerRecyclerViewOne.setLayoutManager(layoutManagerOne);
 
+        pickerRecyclerViewOne.findViewHolderForLayoutPosition(0);
+//        pickerAdapterOne.onBindViewHolder(pickerRecyclerViewOne.findContainingViewHolder(root), 0);
```
Comment (JasperTimm) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java]: More commented code!

File: dhis2-android-app/src/main/java/org/dhis2/mobile/ui/adapters/PickerAdapter.java [hunk 3]
```diff
@@ -95,6 +99,10 @@ public int getItemCount() {
 
         return itemCount;
     }
+    public void setItem(Picker pickerTree){
```
Comment (JasperTimm) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/adapters/PickerAdapter.java]: What's the intention with this? We're making it easier for the user by selecting the first element in the dropdown list of OrgUnits so they don't have to continually select it? If so it should be commented as such.

File: dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java (hunk 2 above)
Comment (zewolfe) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java]: Removed.

File: dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java (hunk 1 above)
Comment (zewolfe) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/fragments/AggregateReportFragment.java]: Removed.

File: dhis2-android-app/src/main/java/org/dhis2/mobile/ui/adapters/PickerAdapter.java (hunk 3 above)
Comment (zewolfe) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/adapters/PickerAdapter.java]: Comment added..

File: dhis2-android-app/src/main/java/org/dhis2/mobile/ui/adapters/PickerAdapter.java [hunk 4]
```diff
@@ -99,6 +99,8 @@ public int getItemCount() {
 
         return itemCount;
     }
+
+    //Sets Picker to first child on the tree.
```
Comment (JasperTimm) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/adapters/PickerAdapter.java]: This adds precisely ZERO. I can read the code to see that's what is happening. Yet I still have no idea why this function exists.

- Rename the method to 'selectFirstItem'.
- Add a comment to the code that calls it explaining why this is a good idea 'most user's only have one orgUnit, select the first item to save them time'.

Comment (zewolfe) [inline dhis2-android-app/src/main/java/org/dhis2/mobile/ui/adapters/PickerAdapter.java]: Changes made.

**Pull request merged**